README.md

```diff
@@ -1,12 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio: mobile version - Contact form
+# Portfolio: Desktop version
 
-> Portfolio: mobile version - Contact form
+> Portfolio: Desktop version
 
```

Comment:
- [ ] There appears to be a problem with your images. They are not broken. Please them

![amr-imgs](https://user-images.githubusercontent.com/36542986/134592374-6ba78688-519f-49b3-a735-f61b6c155d9c.jpg)

- [ ] Your page also appears to be missing the background effects contained in the templates

![amr-bckg](https://user-images.githubusercontent.com/36542986/134592867-60969dc3-ea06-4620-909a-398b99d4a084.jpg)

Reply:
Not sure, I totally understood what you meant here

Reply:
@Amrhub I meant that your project does not have the background images present in the Figma template.